test/test_distributor.js

```
@@ -282,5 +282,339 @@ function runAllTests() {
+        }
+    });
+    
+    if (foundReadme) {
+        console.log(`${COLOR_GREEN}PASS: OTA readme file(s) found${COLOR_RESET}`);
+        testsPassed++;
+    } else {
+        console.log(`${COLOR_YELLOW}SKIP: No OTA readme files found${COLOR_RESET}`);
+    }
+}
+
+function runAllTests() {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'runAllTests'. Consider to delete it or rename it.

'runAllTests' is defined here:


(lint/suspicious/noRedeclare)